kura/org.eclipse.kura.wire.component.provider/META-INF/MANIFEST.MF

```diff
@@ -24,7 +24,7 @@ Import-Package: org.eclipse.kura;version="[1.2,2.0)",
  org.eclipse.kura.type;version="[1.0,2.0)",
  org.eclipse.kura.util.base;version="[1.0,2.0)",
  org.eclipse.kura.util.collection;version="[1.0,2.0)",
- org.eclipse.kura.wire;version="[1.0,1.1)",
+ org.eclipse.kura.wire;version="[1.0,2.0)",
```

Comment:
It does implement WireEmitter and WireReceiver

Reply:
They are @ConsumerType

Reply:
Aah.. you are right.